tsconfig.json

```
@@ -14,17 +15,13 @@
     "noFallthroughCasesInSwitch": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "plugins": [
-      {
-        "name": "nexus/typescript-language-service"
-      }
-    ],
+    "plugins": [{ "name": "nexus/typescript-language-service" }],
     "removeComments": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
-    "target": "ES6"
+    "target": "ES6",
```

```suggestion
    "target": "ES6"
```

think this was a JS linter trying to autoformat this file, but the trailing comma is technically invalid JSON

---

Lol it's was! Thanks for fix it